corehq/apps/export/views/new.py

```diff
@@ -116,10 +115,7 @@ def terminology(self):
     def page_context(self):
         owner_id = self.export_instance.owner_id
         number_of_apps_to_process = 0
-        is_all_case_types_export = (
-            isinstance(self.export_instance, CaseExportInstance)
-            and self._is_bulk_export
-        )
+        is_all_case_types_export = self._is_bulk_case_export
```

Comment:
nit: do we need this variable anymore? Seems like we could just use `self._is_bulk_case_export` directly.